mysql/plan.sh

```diff
@@ -45,6 +45,12 @@ pkg_bin_dirs=(bin)
 pkg_include_dirs=(include)
 pkg_lib_dirs=(lib)
 
+pkg_exports=(
+    [port]=port
+    [pwd]=app_password
```

Comment:
you are totally right. I had used uid and pwd because that's what the connection string provider I was using used for tokens but we should export the full names.